python/ray/tune/schedulers/pbt.py

```diff
@@ -227,9 +244,32 @@ def __init__(self,
         self._num_checkpoints = 0
         self._num_perturbations = 0
 
+    def fill_config(self, config, attr, search_space):
+        # If attribute in hyperparam_mutations but not in config, add to config
+        if callable(search_space):
+            config[attr] = search_space()
+        elif isinstance(search_space, sample_from):
+            config[attr] = search_space.func(None)
+        elif isinstance(search_space, list):
+            config[attr] = random.choice(search_space)
+        elif isinstance(search_space, dict):
+            config[attr] = {}
+            for k, v in search_space.items():
+                self.fill_config(config[attr], k, v)
```

Comment:
can you make this a helper method?

Reply:
Do you mean indicate this method as internal use only, ie `_fill_config`?

Reply:
What I mean is to take it out from the class and define it as a helper method. There's no need to define it in the class, and by definition, it becomes private.

Reply:
ok i removed it from the class